.travis.yml

```diff
@@ -6,8 +6,6 @@ go:
   - master
   - 1.11.x
   - 1.10.x
-  - 1.9.x
```

Comment:
maybe specify in the changelog that we are dropping officially support for 1.9 and 1.8 ? ( but they are still expected to work)